src/com/lftechnology/java/training/sanish/application/dbconnection/DbConnect.java

```diff
@@ -0,0 +1,67 @@
+package com.lftechnology.java.training.sanish.application.dbconnection;
+
+import com.lftechnology.java.training.sanish.application.component.Constants;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Provide database connection and close static functions
+ *
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public abstract class DbConnect {
+    static final String JDBC_DRIVER = "com.mysql.jdbc.Driver";
+    static final String DB_URL = "jdbc:mysql://localhost:3306/service_db?rewriteBatchedStatements=true";
+    static final String USER = "root";
+    static final String PASS = "";
+    private static final Logger LOGGER = Logger.getLogger(DbConnect.class.getName());
+    private static Connection connection;
+
+    public DbConnect() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Hide this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)